source/speechDictHandler.py

```diff
@@ -96,10 +101,17 @@ def sub(self, text):
 		return text
 
 def processText(text):
-	if not globalVars.speechDictionaryProcessing:
-		return text
-	for type in dictTypes:
-		text=dictionaries[type].sub(text)
+	if globalVars.speechDictionaryProcessing:
+		for type in dictTypes:
+			text=dictionaries[type].sub(text)
+	numberSetting = config.conf["speech"]["readNumbersAs"]
+	#0: processes default behavior. 1-3: splits on single-triple digits.
+	if numberSetting == 1:
+		text = RE_SINGLE_DIGITS.sub(r"\1 ", text)
+	if numberSetting == 2:
+		text = RE_DOUBLE_DIGITS.sub(r"\1 ", text)
+	if numberSetting == 3:
+		text = RE_TRIPLE_DIGITS.sub(r"\1 ", text)

```

Comment:
1. The second and third `if` should be `elif` to avoid unnecessary checks.
2. I'd suggest splitting this last little bit into a function like `processNumbers(numberSetting, text)` and calling that from `processText`. While we don't have unit tests now, this piece of code would be highly suitable for isolated unit testing.